nasa-gibs/worldview
# handle empty layers param

## Description

Fixes #3322 

## To test
1. Go to [http://localhost:3000/?l=&now=2013-07-01T12](http://localhost:3000/?l=&now=2013-07-01T12)
2. Confirm page loads without errors, no layers present
